packages/ui/fe-bundle/src/pages/[host]/[[...slug]].tsx

```diff
@@ -20,40 +21,40 @@ export declare namespace Docs {
     export interface Props {
         docs: FernRegistryDocsReadV2.LoadDocsForUrlResponse;
         resolvedUrlPath: ResolvedUrlPath;
-        typographyStyleSheet?: string;
+        typographyStyleSheet: string | undefined;
+        backgroundImageStyleSheet: string | undefined;
         nextPath: ResolvedUrlPath | null;
         previousPath: ResolvedUrlPath | null;
     }
 }
 
-function Typography({ stylesheet }: { stylesheet: string }) {
-    return (
-        // eslint-disable-next-line react/no-unknown-property
-        <style id="fern-font-stylesheet" jsx global>
-            {stylesheet}
-        </style>
-    );
-}
-
 export default function Docs({
     docs,
-    typographyStyleSheet,
+    typographyStyleSheet = "",
+    backgroundImageStyleSheet = "",
     resolvedUrlPath,
     nextPath,
     previousPath,
 }: Docs.Props): JSX.Element {
     return (
         <>
-            {typographyStyleSheet != null && <Typography stylesheet={typographyStyleSheet} />}
             <main className={classNames(inter.className, "typography-font-body")}>
+                {/* 
+                    We concatenate all global styles into a single instance,
+                    as styled JSX will only create one instance of global styles
+                    for each component.
+                */}
+                {/* eslint-disable-next-line react/no-unknown-property */}
+                <style jsx global>
+                    {`
+                        ${typographyStyleSheet}
+                        ${backgroundImageStyleSheet}
+                    `}
+                </style>
```

Comment:
looks great! just curious, would this work?

```tsx
                <style jsx global>
                    {typographyStyleSheet}
                    {backgroundImageStyleSheet}
                </style>
```

and then we don't need the `= ""` in the function signature